<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codification</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupParameters">
      <property name="title">
       <string>Generals parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QComboBox" name="mComboParameterSeparator">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>*</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>,</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Parameters separator</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="mComboCodeSeparator">
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>*</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>,</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Codes separator</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupCodification">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Codification</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="1">
        <widget class="QgsMapLayerComboBox" name="mLayerOutput"/>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="mSaveCode">
         <property name="text">
          <string>Save the cade</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="mDeleteCode">
         <property name="text">
          <string>Delete the code</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Output layer:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="mGeometry"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Geometry:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mDescription"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="mComboCode">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Code:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsFileWidget" name="mGeoPackage" native="true">
         <property name="filter" stdset="0">
          <string>*.gpkg</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>GeoPackage:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Attributes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>361</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="mAddColumnPushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/plugins/LandSurveyFieldCodes/images/symbologyAdd.png</normaloff>:/plugins/LandSurveyFieldCodes/images/symbologyAdd.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mRemoveColumnPushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/plugins/LandSurveyFieldCodes/images/symbologyRemove.png</normaloff>:/plugins/LandSurveyFieldCodes/images/symbologyRemove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mEmptyColumnsPushButton">
              <property name="text">
               <string>Empty</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QTableWidget" name="mTableWidget">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <column>
             <property name="text">
              <string>Attributes</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Expressions</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
         <zorder>horizontalSpacer</zorder>
         <zorder>mTableWidget</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QgsCollapsibleGroupBox" name="groupSepcialsPoints">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Specials points</string>
      </property>
      <property name="collapsed" stdset="0">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="mAllPoints">
         <property name="text">
          <string>All points</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsFileWidget" name="mGeoPackageAllPoints" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="filter" stdset="0">
          <string>*.gpkg</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QgsMapLayerComboBox" name="mLayerAllPoints">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="mErrorPoints">
         <property name="text">
          <string>Errors points</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsFileWidget" name="mGeoPackageErrorPoints" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="filter" stdset="0">
          <string>*.gpkg</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QgsMapLayerComboBox" name="mLayerErrorPoints">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>15</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Nouveau">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menu_Nouveau"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
